### PR TITLE
update package config, use the same reference for cash and expense

### PR DESCRIPTION
**Pull request by osarze**

### CODE REVIEW

File: src/Http/Controllers/Admin/TripsController.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Credpal\Expense\Models\Trips;
+
+class TripsController extends Controller
+{
+    public function index()
+    {
+        dd("Amdin index");
```
Comment (dharmykoya) [inline src/Http/Controllers/Admin/TripsController.php:13]: You forgot this here

File: src/Http/Requests/Trips/BookTicketRequest.php [hunk 2]
```diff
@@ -26,9 +30,17 @@ public function rules()
         $confirmTicketRequest = new ConfirmTicketPriceRequest();
         $rules = array_merge($confirmTicketRequest->rules(), [
             'amount' => ['required', 'regex:/^\d+(\.\d{1,2})?$/'],
-            'wallet_type' => ['required', 'in:debit,credit'],
-            'wallet_id' => ['required'],
-            'user_id' => ['required', 'exists:users,id']
+            'payment_method' => ['required', Rule::in(ExpenseProcess::PAYMENT_METHOD_CASH, ExpenseProcess::PAYMENT_METHOD_CREDIT_CARD)],
+            'wallet_type' => ['required', Rule::in(Enum::DEBIT, Enum::CREDIT)],
```
Comment (dharmykoya) [inline src/Http/Requests/Trips/BookTicketRequest.php:34]: I thought we were changing wallet type to something else?

File: src/routes/api.php [hunk 3]
```diff
@@ -49,6 +50,15 @@
     Route::post('electricity-request', 'BaxiController@electricityRequest');
 });
 
+/** ========== Admin Expense Route ========= **/
+Route::group([
```
Comment (dharmykoya) [inline src/routes/api.php:54]: Don't you think we should create another file for admin routes?

File: src/Config/expense.php [hunk 4]
```diff
@@ -3,13 +3,16 @@
...
-    'debit_wallet_url' => env('DEBIT_WALLET_TRANSACTION_URL'),
-    'debit_wallet_finalize_url' => env('DEBIT_WALLET_FINALIZE_URL'),
-    'credit_wallet_url' => env('CREDIT_WALLET_TRANSACTION_URL'),
-    'credit_wallet_finalize__url' => env('CREDIT_WALLET_FINALIZE_URL'),
-    'transfer_url' => env('EXPENSE_TRANSFER_URL'),
-    'bills_url' => env('EXPENSE_BILLS_URL'),
-    'wallet_private_key' => env('WALLET_PRIVATE_KEY'),
-    'expense_private_key' => env('EXPENSE_SERVICE_PRIVATE_KEY'),
-    'trips_url' => env('EXPENSE_TRIPS_URL'),
+   'bills_url' => env('EXPENSE_BILLS_URL'),
```
Comment (dharmykoya) [inline src/Config/expense.php:6]: You still left bills url here

**Review by dharmykoya — CHANGES_REQUESTED** [review]

File: src/Http/Requests/Trips/BookTicketRequest.php (hunk 2 above)
Comment (osarze) [reply src/Http/Requests/Trips/BookTicketRequest.php:34]: I and @Geoslim we decided to leave it there for now. If going forward we see that we don't really need it we would then remove it totally

File: src/routes/api.php (hunk 3 above)
Comment (osarze) [reply src/routes/api.php:54]: Hnmnmnmn, That is also not bad

File: src/Config/expense.php (hunk 4 above)
Comment (osarze) [reply src/Config/expense.php:6]: I want @Geoslim to remove it himself since he has already given wisdom, so there wouldn't error

**Review by Geoslim — APPROVED** [review]

**Pull request merged**